src/main/java/com/couchbase/lite/DatabaseUpgrade.java

```diff
@@ -167,11 +173,16 @@ protected void deleteSQLiteFiles() {
     }
 
     private boolean moveAttachmentsDir() {
+        // v1.1.x or v1.2.x
         File oldAttachmentsPath = new File(FileDirUtils.getPathWithoutExt(path) + " attachments");
-        if (!oldAttachmentsPath.canRead())
-            return true;
+        if (!oldAttachmentsPath.exists() || !oldAttachmentsPath.canRead()) {
+            // v1.0.x
+            oldAttachmentsPath = new File(FileDirUtils.getPathWithoutExt(path), "attachments");
+            if (!oldAttachmentsPath.exists() || !oldAttachmentsPath.canRead())
+                return false;
```

Comment:
Even though i merged the PR, can u confirm that return fase is correct. I don't know why previously return true.